src/script/externalRoute.ts

```diff
@@ -20,95 +20,46 @@
 import {currentLanguage} from './auth/localeConfig';
 import {Config} from './Config';
 
-const env = window.wire.env;
-
-export const URL = {
-  ACCOUNT: env.URL?.ACCOUNT_BASE,
-  PRIVACY_POLICY: env.URL?.PRIVACY_POLICY,
-  SUPPORT: env.URL?.SUPPORT.INDEX,
-  TEAM_SETTINGS: env.URL?.TEAMS_BASE,
-  TERMS_OF_USE_PERSONAL: env.URL?.TERMS_OF_USE_PERSONAL,
-  TERMS_OF_USE_TEAMS: env.URL?.TERMS_OF_USE_TEAMS,
-  WEBAPP: {
-    INTERNAL: 'https://wire-webapp-staging.wire.com',
-    PRODUCTION: env.APP_BASE || 'https://app.wire.com',
-    STAGING: 'https://wire-webapp-staging.zinfra.io',
-  },
-  WEBSITE: env.URL?.WEBSITE_BASE,
-};
-
-export const URL_PATH = {
-  CREATE_TEAM: '/create-team/',
-  DECRYPT_ERROR_1: '/articles/207948115',
-  DECRYPT_ERROR_2: '/privacy/error-2/',
-  MANAGE_SERVICES: '/services/',
-  MANAGE_TEAM: '/login/',
-  PASSWORD_RESET: '/forgot/',
-  PRIVACY_HOW: '/privacy/how/',
-  PRIVACY_UNVERIFIED_USERS: '/articles/202857164',
-  PRIVACY_WHY: '/articles/207859815',
-  SUPPORT_USERNAME: '/support/username/',
-} as const;
```

Comment:
some of those are removed as deprecated
support links still in use have been moved with existing ones as environment variables instead of being hardcoded
url paths intended to be dynamically appended have been made into the `URL_SUBPATH` env variables as to be end user configurable